glue_jupyter/bqplot/image/frb_mark.py

```diff
@@ -0,0 +1,68 @@
+# This is an image sub-class that automatically gets its values from an object
+# that implements __call__ with a bounds= argument. It is the equivalent of
+# FRBArtist in glue-core.
```

Comment:
What does FRB stand for?

Reply:
Fixed resolution buffer - I'll write it out in the comments